website/content/en/preview/concepts/nodeclasses.md

```diff
@@ -79,6 +79,11 @@ spec:
     - name: my-ami
     - id: ami-123
 
+    # Automatically select the latest EKS optimzed AMI for the given AMI family.
+    # Note: This term is mutually exclusive and may not be specified with other AMI selector terms
```

Comment:
Let's be more specific here in the notes as well so people read it about the dangers of enabling this in production. I know it's elsewhere, but we should put it everywhere we can.